ctapipe/io/metadata.py

```diff
@@ -61,9 +60,25 @@ class Product(HasTraits):
     description = Unicode("unknown")
     creation_time = AstroTime()
     id_ = Unicode(help="leave unspecified to automatically generate a UUID")
-    data_category = Enum(["S", "A", "B", "C", "Other"], "Other")
-    data_level = Enum(
-        ["R0", "R1", "DL0", "DL1", "DL2", "DL3", "DL4", "DL5", "DL6", "Other"], "Other"
+    data_category = Enum(["Sim", "A", "B", "C", "Other"], "Other")
+    data_level = List(
+        Enum(
```

Comment:
We already have the `ctapipe.io.DataLevel` enum. Would that be usable here?

It is an int enum, maybe we should change to a string enum and then we could just use it here

Reply:
This should work: `datalevel=List(Enum([level.name for level in DataLevel]))`